apps/frontend/src/app/poll/ical/ical.component.html

```diff
@@ -0,0 +1,74 @@
+<ngbx-modal #modal [back]="['..']">
+  <ng-container modal-title>
+    iCal Export
+  </ng-container>
+  <ng-container modal-body>
+    <h5>
+      Options
+    </h5>
+    <ngbx-validator-form [model]="config"></ngbx-validator-form>
+    <h5>
+      Preview
+    </h5>
+    <div class="card">
+      <div class="card-body">
+        <table>
+          <tr>
+            <th>
+              <i class="bi-square-fill text-primary-emphasis"></i>
+            </th>
+            <td>
+              <h5 class="card-title">
+                {{ config.customTitle ?? poll?.title }}:
+                {{ exampleEvent?._participants?.[0]?.name }}
```

Comment:
```suggestion
```
Imo it's weird having the name of the first participant in the title.

Reply:
This is intentional. With a 1-1 poll you immediately see in your calendar who will show up in the next slot. It was very useful for the STC presentations.